roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
+
+- name: setup swift rings - scaleup
+  command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} scaleup
+           --iterations {{ swift_ring.iterations }} 
+  async: "{{ swift_ring.iterations * 60 * 60|int }}"
+  poll: 0
```

Comment:
A poll 0 here means that we won't wait for this task to complete, nor will we check back in on it. It's just fire and forget and move on with life. As such, putting in a specific value to the `async` line doesn't make sense.